.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,77 @@
+name: Publish PelicanFS distribution to PyPI
+on: 
+  release:
+    types:
+      - published
+
+jobs:
+  publish-to-pypi:
+    if: "!github.event.release.prerelease"
+    name: >-
+      Publish Python Distribution to PyPI
+    needs:
+    - build
+    runs-on: ubuntu-latest
+    environment:
+      name: pypi
+      url: https://pypi.org/p/pelicanfs
+    permissions:
+      id-token: write
+    steps:
+    - name: Download all the dists
+      uses: actions/downaload-artifact@v3
+      with:
+        name: python-package-distributions
+        path: dist/
+    - name: Publish distribution to PyPI
+      uses: pypa/gh-action-pypi-publish@release/v1
+
+  github-release:
+    name: >-
+      Sign the Python distribution with Sigstore
+      and upload them to GitHub Release
+    needs:
+    - publish-to-pypi
+    runs-on: ubuntu-latest
```

Comment:
This may be an ignorant question on my part, but it's sometimes frustrating when PyPi has wheels for x86 but not for arm. Do we get packages for both architectures out of this? If not, is there a way to publish for both?

Reply:
Since the code is pure python, it isn't architecture or processor specific. The wheel that it generates is good for both. It just that the build/publish action needs a linux container to run in.